src/main/java/consoleforgui/HelperConsole.java

```diff
@@ -12,7 +12,7 @@
  * by ExistingUserController and NewUserController
  */
 public class HelperConsole {
-
+// TODO: several methods are never used (or the methods calling them are never used), refactor if time allows.
```

Comment:
Yeah some of the methods were only used for in line commands, will definitely need to delete after GUI fully finished!